docker/nginx/nginx.conf

```diff
@@ -0,0 +1,66 @@
+upstream graph-node {
+    server graph-node:8000;
```

Comment:
we don't want to expose all these extra ports, its unsafe

Reply:
Originally I had kept just 800 but after last night's call I changed it to include all. nvm Ill changed it back to the original.